### PR TITLE
[PLAYER-13] adding showAdCountDown parameter to show/hide remaining time for ads

### PR DESCRIPTION
**Pull request by ksunika**
should work with https://github.com/ooyala/skin-config/pull/157

### CODE REVIEW

File: js/components/adPanel.js [hunk 1]
```diff
@@ -74,15 +74,17 @@ var AdPanel = React.createClass({
...
-    } else {
-      remainingTime = parseInt(this.props.adVideoDuration - this.props.currentPlayhead)
+    if (this.props.skinConfig.adScreen.showAdCountDown) {
+      var remainingTime;
+      if (isLive) {
+        remainingTime = parseInt((this.props.adStartTime + this.props.adVideoDuration * 1000 - new Date().getTime())/1000);
+      } else {
+        remainingTime = parseInt(this.props.adVideoDuration - this.props.currentPlayhead)
+      }
+      remainingTime = Utils.formatSeconds(Math.max(0, remainingTime));
```
Comment (pilievOoyala) [inline js/components/adPanel.js]: we should probably have a check to make sure we have a finite number. Then we can avoid possible showing NaN on the screen.

Comment (pilievOoyala) [inline js/components/adPanel.js]: Also, add a debug log statement if the data is bad.

**Comment by ksunika** [comment]
@pilievOoyala, I addressed your comments. For some reason I can't point the config file to the one used in the corresponding pull request in skin config. If I try, I get conflicts. Not sure how to address that.

**Comment by mistermendez** [comment]
@ksunika I had to sync with latest config, then merge master into this branch.

Good to go.

**Comment by pilievOoyala** [comment]
Looks good. Merging!

**Pull request merged**